student.rb

```diff
@@ -1,8 +1,15 @@
 require_relative './person'
```

Comment:
- [ ] It would be better if we add instructions about how to clone and run the app locally to our README file so that non-technical people can appreciate it :+1: